<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QFitParCfgWidget</class>
  <widget class="QWidget">
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>330</width>
        <height>317</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>QFitParCfgWidget</string>
    </property>
    <widget class="QLabel" name="TextLabel6" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>10</y>
          <width>230</width>
          <height>20</height>
        </rect>
      </property>
      <property name="text" >
        <string>Full parameter name</string>
      </property>
      <property name="wordWrap" >
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QLabel" name="TextLabel7" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>100</y>
          <width>30</width>
          <height>20</height>
        </rect>
      </property>
      <property name="text" >
        <string>Min</string>
      </property>
      <property name="wordWrap" >
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QLabel" name="TextLabel8" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>140</y>
          <width>35</width>
          <height>20</height>
        </rect>
      </property>
      <property name="text" >
        <string>Max</string>
      </property>
      <property name="wordWrap" >
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QLabel" name="TextLabel9" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>190</y>
          <width>60</width>
          <height>20</height>
        </rect>
      </property>
      <property name="text" >
        <string>Epsilon</string>
      </property>
      <property name="wordWrap" >
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="ParNameEdit" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>30</y>
          <width>200</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="ParFixedChk" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>70</y>
          <width>160</width>
          <height>23</height>
        </rect>
      </property>
      <property name="text" >
        <string>parameter fixed</string>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="RangeMinEdit" >
      <property name="geometry" >
        <rect>
          <x>50</x>
          <y>100</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="RangeMaxEdit" >
      <property name="geometry" >
        <rect>
          <x>50</x>
          <y>140</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="EpsilonEdit" >
      <property name="geometry" >
        <rect>
          <x>70</x>
          <y>190</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <includes>
    <include location="local">QGo4LineEdit.h</include>
  </includes>
</ui>
